### ISSUE TITLE
0.17 "desert" is not a valid autoplace control name

### ISSUE THREAD

**Issue by Webatron11** [Oarcinae/FactorioScenarioMultiplayerSpawn#92]
> Error while running event level::on_init()
"desert" is not a valid autoplace control name.
stack traceback: <

  >  ...ng/Factorio/temp/currently-playing/locale/oarc_utils.lua:1210: in function 'CreateGameSurface'
    ...Data/Roaming/Factorio/temp/currently-playing/control.lua:103: in function <...Data/Roaming/Factorio/temp/currently-playing/control.lua:91>
stack traceback:
    [C]: in function 'create_surface'
    ...ng/Factorio/temp/currently-playing/locale/oarc_utils.lua:1210: in function 'CreateGameSurface'
    ...Data/Roaming/Factorio/temp/currently-playing/control.lua:103: in function <...Data/Roaming/Factorio/temp/currently-playing/control.lua:91> <

I get this when tryin to start the scenario with **no** mods

**Comment by Oarcinae** [comment]
You probably pulled the master branch instead of dev_0.17

**Comment by Webatron11** [comment]
I used dev_0.17

**Comment by Oarcinae** [comment]
You sure?
`locale/oarc_utils.lua:1210` from your error suggests otherwise.

**Comment by Webatron11** [comment]
I am just downloading the zip. I configured everything, and it still won't work.

**Comment by Oarcinae** [comment]
Meaning still the same error? Or something else now?

**Comment by Webatron11** [comment]
same error

**Comment by Oarcinae** [comment]
If your download has a "locale" folder, you're not pulling the right branch.

**Comment by Webatron11** [comment]
ok, I will try some more stuff out

**Comment by Oarcinae** [comment]
https://github.com/Oarcinae/FactorioScenarioMultiplayerSpawn/tree/dev_0.17

**Comment by Webatron11** [comment]
ok, I will use this.

**Comment by Oarcinae** [comment]
Possibly this direct link will work too https://github.com/Oarcinae/FactorioScenarioMultiplayerSpawn/archive/dev_0.17.zip

**Comment by Webatron11** [comment]
I got it working with no mods. I am planning on playing with mods though. RSO (the actual mod) is causing a problem, so should disabling the softmod on the scenario fix it?

**Comment by Webatron11** [comment]
Nevermind. I cant find an option to disable the RSO softmod on the scenario config. I also have fixed the error, so i am closing the issue. 

**TLDR:**
Make sure you are using the latest and greatest version of the scenario, and eveything should work out.
